avaframe/com1DFAPy/com1DFA.py

```diff
@@ -214,49 +214,31 @@ def getSimulation(cfg, rel, entResInfo):
     log.info('Release area scenario: %s - perform simulations' % (relName))
 
     # define simulation type
-    cuSim = []
-    for simType in simTypeList:
-        if simType == 'ent':
-            if entResInfo['flagEnt'] == 'Yes':
-                cuSim.append('ent')
-            else:
-                log.error('No entrainment area available for ent simulation')
-                raise FileNotFoundError
-        elif simType == 'entres':
-            if entResInfo['flagEnt'] == 'Yes' and entResInfo['flagRes'] == 'Yes':
-                cuSim.append('entres')
-            else:
-                if entResInfo['flagEnt'] == 'Yes' and entResInfo['flagRes'] == 'No':
-                    log.error('No resistance area available for entres simulation')
-                    raise FileNotFoundError
-                elif entResInfo['flagEnt'] == 'No' and entResInfo['flagRes'] == 'Yes':
-                    log.error('No entrainment area available for entres simulation')
-                    raise FileNotFoundError
-                elif entResInfo['flagEnt'] == 'No' and entResInfo['flagRes'] == 'No':
-                    log.error('No resistance and no entrainment area available for entres simulation')
-                    raise FileNotFoundError
-        elif simType == 'res':
-            if entResInfo['flagRes'] == 'Yes':
-                cuSim.append('res')
-            else:
-                log.error('No resistance area available for res simulation')
-                raise FileNotFoundError
-        elif simType == 'null':
-            cuSim.append('null')
-        elif simType == 'available':
-            if entResInfo['flagEnt'] == 'Yes' and entResInfo['flagRes'] == 'Yes':
-                cuSim.append('entres')
-            elif entResInfo['flagEnt'] == 'Yes' and entResInfo['flagRes'] == 'No':
-                cuSim.append('ent')
-            elif entResInfo['flagEnt'] == 'No' and entResInfo['flagRes'] == 'Yes':
-                cuSim.append('res')
-            elif entResInfo['flagEnt'] == 'No' and entResInfo['flagRes'] == 'No':
-                cuSim.append('null')
-
-    # remove duplicate simulation types 
-    cuSim = set(cuSim)
-
-    return relName, cuSim, relDict, badName
+    if 'available' in simTypeList:
+        if entResInfo['flagEnt'] == 'Yes' and entResInfo['flagRes'] == 'Yes':
+            simTypeList.append('entres')
+        elif entResInfo['flagEnt'] == 'Yes' and entResInfo['flagRes'] == 'No':
+            simTypeList.append('ent')
+        elif entResInfo['flagEnt'] == 'No' and entResInfo['flagRes'] == 'Yes':
+            simTypeList.append('res')
+        elif entResInfo['flagEnt'] == 'No' and entResInfo['flagRes'] == 'No':
+            simTypeList.append('null')
```

Comment:
This should be added to the simTypeList always, as discussed

Reply:
I added it in the case of 'available', so that for dev purposes if just 'ent' is specified the 'null' is not added

Reply:
jup, that's fine, so if just 
`ent` 
is specified, it should do no `nul`l. However if just 
`available`
is specified, it should add the 'null' simulation. 
This still means all option are available for dev purposes